101-cses-multi-role-rdp/readme.md

```diff
@@ -0,0 +1,17 @@
+# Cloud Services (extended support) template
```

Comment:
change the casing on the filename to: README.md

Also see the pipeline results for the proper links